XTS/recovery/src/org/jboss/jbossts/xts/recovery/participant/ba/BAParticipantRecoveryModule.java

```diff
@@ -166,7 +167,11 @@ private void doRecoverParticipant( Uid recoverUid )
                         } catch (IllegalAccessException iae) {
                             // this is another configuration problem, log an error
                             RecoveryLogger.i18NLogger.error_participant_ba_BAParticipantRecoveryModule_5(participantRecordClazzName, recoverUid, iae);
-                        }
+                        } catch (InvocationTargetException e) {
+                            RecoveryLogger.i18NLogger.error_participant_ba_BAParticipantRecoveryModule_5(participantRecordClazzName, recoverUid, e);
+			} catch (NoSuchMethodException e) {
+                            RecoveryLogger.i18NLogger.error_participant_ba_BAParticipantRecoveryModule_5(participantRecordClazzName, recoverUid, e);
+			}
```

Comment:
No sure the same logging message can be used for these two exceptions

Reply:
@jmfinelli As discussed above for ProtocolManager class here also i think we can use the same message `error_participant_ba_BAParticipantRecoveryModule_5` for all 4 Exception

Reply:
Could you please group similar catch cases together? Thanks